### PR TITLE
update/add widgets, product reset button

### PR DESCRIPTION
**Pull request by bfocht**
This is the version that was release in beta a few days ago.  
this release 

- updates the domains search widget, product widget.
- adds a login widget
- adds shortcodes
- add a product reset widget
- increases unit test code coverage

More details about the changes are in this video https://youtu.be/RlyB8WvJmbY

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 **Tags:**              [admin](https://wordpress.org/plugins/tags/admin/), [posts](https://wordpress.org/plugins/tags/posts/), [users](https://wordpress.org/plugins/tags/users/)  
 **Requires at least:** 4.6  
 **Tested up to:**      4.8  
-**Stable tag:**        1.0.0  
+**Stable tag:**        0.2.0
```
Comment (cberesford) [inline readme.md]: Should this be 1.2.0?

File: readme.md [hunk 2]
```diff
@@ -46,8 +46,70 @@ You can call our support team at (480) 505-8857
...
+We encourage each reseller to create their own unique site and content for their storefronts.  This provides a better experience for your customers and more opportunities for search engine optimization.
+While we can't test every theme with our plugin, we try our best to make the plugin compatible.  GoDaddy provides several free themes including [Primer](https://wordpress.org/themes/primer/) and [Activation](https://wordpress.org/themes/activation/) that we test with our plugin.  You can use content from the turnkey website to help you build your own WordPress site.
+
+### I am getting an error while trying to activate the plugin. What should I do? ###
+If you are having an issue, we recommend you deactivate your other plugins and then try to activate the Reseller Store plugin.  You can also try changing to another theme and the try re activating the plugin.  Before making any of these changes, we recommend you make a backup of your site and database.  
+
+While we strive to make the plugin compatible many themes and plugins, our plugin creates a custom post type and some themes and custom builder plugins can interfere with the registration of the custom post type.
```
Comment (cberesford) [inline readme.md]: While we strive to make the plugin compatible **with** many themes and plugins, our plugin creates a custom post type, **which can cause** ~~and~~ some themes and custom builder plugins **to** ~~can~~ interfere with the registration of the custom post type.

File: readme.md [hunk 3]
```diff
@@ -46,8 +46,70 @@ You can call our support team at (480) 505-8857
...
+
+### What shortcodes are supported? ###
+While we recommend  you use our Widgets for your storefront, we do have a shortcodes for each widget.
```
Comment (cberesford) [inline readme.md]: Widgets --> widgets

File: readme.md [hunk 4]
```diff
@@ -46,8 +46,70 @@ You can call our support team at (480) 505-8857
...
+```
+#### Log in/out Button
+```
+[rstore_login welcome_message="Welcome Back" login_button_text="Log In" logout_button_text="Log Out" ]
+```
+
+
 ## Changelog ##
 
+### NEXT - September 2017 ###
```
Comment (cberesford) [inline readme.md]: November 2017

File: readme.txt [hunk 5]
```diff
@@ -3,7 +3,7 @@ Contributors:      godaddy, fjarrett, bfocht, eherman24
 Tags:              admin, posts, users
 Requires at least: 4.6
 Tested up to:      4.8
-Stable tag:        1.0.0
+Stable tag:        0.2.0
```
Comment (cberesford) [inline readme.txt]: 1.2.0?

File: readme.txt [hunk 6]
```diff
@@ -42,8 +42,70 @@ You can call our support team at (480) 505-8857
...
+```
+#### Log in/out Button
+```
+[rstore_login welcome_message="Welcome Back" login_button_text="Log In" logout_button_text="Log Out" ]
+```
+
+
 == Changelog ==
 
+= NEXT - September 2017 =
```
Comment (cberesford) [inline readme.txt]: See comments in `readme.md` - they apply here as well

File: reseller-store.php [hunk 7]
```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: GoDaddy Reseller Store
  * Description: Sell hosting, domains, and more right from your WordPress site.
- * Version: 1.0.0
+ * Version: 0.2.0
```
Comment (cberesford) [inline reseller-store.php]: 1.2.0

File: reseller-store.php [hunk 8]
```diff
@@ -40,7 +40,7 @@
 	 *
 	 * @var string
 	 */
-	const VERSION = '1.0.0';
+	const VERSION = '0.2.0';
```
Comment (cberesford) [inline reseller-store.php]: 1.2.0

File: includes/class-api.php [hunk 9]
```diff
@@ -77,60 +77,62 @@ public function __construct() {
...
 
 	}
 
+
 	/**
-	 * Add required query args to a given URL.
+	 * Build a SSO login or logout url.
 	 *
-	 * @since 0.2.0
+	 * @since NEXT
```
[the remark this answers is not in the document]
Comment (bfocht) [reply includes/class-api.php]: yes this is correct.  running `npm version` will automagically update these

File: includes/butterbean/controls/index.php [hunk 10]
```diff
@@ -0,0 +1,4 @@
+<?php
+/**
```
[the remark this answers is not in the document]
Comment (bfocht) [reply includes/butterbean/controls/index.php]: yes

File: readme.md (hunk 1 above)
Comment (bfocht) [reply readme.md]: see @EvanHerman commit https://github.com/godaddy/wp-reseller-store/pull/42/commits/47242ec0bfa12d3aabe9ebdd976c33846dd82655

File: readme.txt (hunk 5 above)
Comment (bfocht) [reply readme.txt]: see commit https://github.com/godaddy/wp-reseller-store/pull/42/commits/47242ec0bfa12d3aabe9ebdd976c33846dd82655

File: reseller-store.php (hunk 8 above)
Comment (bfocht) [reply reseller-store.php]: there is no such thing as version 1.2.0 see releases https://github.com/godaddy/wp-reseller-store/releases

File: readme.txt [hunk 11]
```diff
@@ -45,28 +45,28 @@ Development of this plugin is done on [GitHub](https://github.com/godaddy/wp-re
...
 
 = I am getting an error while trying to activate the plugin. What should I do? =
-If you are having an issue, we recommend you deactivate your other plugins and then try to activate the Reseller Store plugin.  You can also try changing to another theme and the try re activating the plugin.  Before making any of these changes, we recommend you make a backup of your site and database.  
+If you are having an issue, we recommend you deactivate your other plugins and then try to activate the Reseller Store plugin. You can also try changing to another theme and the try re activating the plugin. Before making any of these changes, we recommend you make a backup of your site and database.
 
-While we strive to make the plugin compatible many themes and plugins, our plugin creates a custom post type and some themes and custom builder plugins can interfere with the registration of the custom post type.
+While we strive to make the plugin compatible many themes and plugins, our plugin creates a custom post type which can cause some themes and custom builder plugins to interfere with the registration of the custom post type.
```
Comment (cberesford) [inline readme.txt]: While we strive to make the plugin compatible **WITH** many themes ;)

File: .eslintignore [hunk 12]
```diff
@@ -0,0 +1,2 @@
+*.min.js
+js-cookie.js
```
Comment (cberesford) [inline .eslintignore]: newline

**Pull request merged**